lib/proto/attach-link.js

```diff
@@ -51,10 +58,9 @@ var linkAction = function(el, event) {
     this.reload()
     return
   }
-  this.options.requestOptions = this.options.requestOptions || {};
```

Comment:
I didn't see anywhere else in the code `requestOptions` would have been modified from its default of `{}`, so I moved its instantiation to the top to be consistent with `attach-form.js`.